tests/target/existential_type.rs

```diff
@@ -6,10 +6,18 @@ pub type Adder<F, T>
 where
     T: Clone,
     F: Copy,
-= Fn(T) -> T;
```

Comment:
I guess this 'impl' should not be stripped. The compile result seems to be changed.
example is [here](https://play.rust-lang.org/?version=nightly&mode=debug&edition=2018&gist=27116dcfcb799c5bcba4e48f1af6a0a0)